scripts/examples/50-OpenMV-Boards/50-STM32-Boards/50-Board-Control/pwm_control.py

```diff
@@ -10,10 +10,9 @@
 from pyb import Pin, Timer
 
 tim = Timer(4, freq=1000)  # Frequency in Hz
-# Generate a 1KHz square wave on TIM4 with 50%, 75% and 50% duty cycles on channels 1, 2 and 3 respectively.
+# Generate a 1KHz square wave on TIM4 with 50%, 75% and 50% duty cycles on channels 1, 2 respectively.
 ch1 = tim.channel(1, Timer.PWM, pin=Pin("P7"), pulse_width_percent=50)
 ch2 = tim.channel(2, Timer.PWM, pin=Pin("P8"), pulse_width_percent=75)
-ch3 = tim.channel(3, Timer.PWM, pin=Pin("P9"), pulse_width_percent=50)
```

Comment:
But why remove channel 3?

Reply:
Because there are only 2 channels on the OpenMV H7 Plus.